canvas_account_admin_tools/templatetags/django_helpers.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+from django.conf import settings
+from django.utils import timezone
+from django.utils.safestring import mark_safe
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def settings_value(name):
+    return getattr(settings, name, "")
+
+
+@register.filter(is_safe=True)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It looks like you have set the `is_safe` flag to `True` for a custom filter. The `is_safe` flag tells Django not to apply escaping on the value returned by the filter even though the input is escaped. This could expose your application to cross-site scripting (XSS) vulnerabilities. We recommend that you remove `is_safe` and use `mark_safe()` to explicitly mark safe content.

[Learn more](https://cwe.mitre.org/data/definitions/79.html)